src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  try {
+    await execaCommand(
+      'pnpm stylelint **/*.{css,scss,less,js,tsx} --max-warnings 0',
```

Comment:
maybe double quotes are good around the pattern for Bash compatibility - they also use them in the docs: https://stylelint.io/user-guide/get-started/#:~:text=And%20then%20run%20Stylelint%20on%20both%20your%20CSS%20and%20JavaScript%20files%3A

Reply:
'pnpm stylelint "**/*.{css,scss,less,js,tsx}" --max-warnings 0',

when i use the "" it is failing

```
await execaCommand(
      'pnpm stylelint "**/*.{css,scss,less,js,tsx}" --max-warnings 0',
    );

Error: No files matching the pattern ""**/*.{css,scss,less,js,tsx}"" were found.
```